### PR TITLE
CAMEROON: Declarant/informant page in Birth form

### PR DESCRIPTION
**Pull request by wilrona**
Cameroon config : https://github.com/agence-digital/opencrvs-cameroon/pull/66

### CODE REVIEW

File: packages/client/src/forms/configuration/default/birth.ts [hunk 1]
```diff
@@ -978,13 +978,13 @@ export const birthRegisterForms: ISerializedForm = {
...
-                "(draftData && draftData.registration && draftData.registration.informantType && selectedInformantAndContactType.selectedInformantType && (selectedInformantAndContactType.selectedInformantType === 'MOTHER' || selectedInformantAndContactType.selectedInformantType === 'FATHER'))"
-            }
-          ],
+          // conditionals: [
+          //   {
+          //     action: 'hide',
+          //     expression:
+          //       "(draftData && draftData.registration && draftData.registration.informantType && selectedInformantAndContactType.selectedInformantType && (selectedInformantAndContactType.selectedInformantType === 'MOTHER' || selectedInformantAndContactType.selectedInformantType === 'FATHER'))"
+          //   }
+          // ],
```
Comment (rikukissa) [inline packages/client/src/forms/configuration/default/birth.ts:987]:
```suggestion
```

**Review by rikukissa — APPROVED** [review]

File: packages/client/src/views/PrintCertificate/PDFUtils.ts [hunk 2]
```diff
@@ -70,6 +70,28 @@ export function executeHandlebarsTemplate(
   templateString: string,
   data: Record<string, any> = {}
 ): string {
+  Handlebars.registerHelper('ifCond', function (v1, operator, v2, options) {
```
Comment (rikukissa) [inline packages/client/src/views/PrintCertificate/PDFUtils.ts:73]:
```
Error: @opencrvs/client: src/views/PrintCertificate/PDFUtils.ts(76,39): error TS2683: 'this' implicitly has type 'any' because it does not have a type annotation.
```
compiler failing here. Verify that `yarn test:compilation` is passing in `packages/client` after you fix this

File: packages/client/src/views/PrintCertificate/ReviewCertificateAction.tsx [hunk 3]
```diff
@@ -360,6 +360,7 @@ function mapStatetoProps(
         ...draft.data,
         template: {
           ...draft.data.template,
+          informantType: draft.data.informant.relationship,
```
Comment (rikukissa) [inline packages/client/src/views/PrintCertificate/ReviewCertificateAction.tsx:363]: Hey! To do this as per the convention, this should be done in `default/birth.ts` inside the mapping block here:
https://github.com/opencrvs/opencrvs-core/blob/fd52441d619d69319d775006bae52ce0799390db/packages/client/src/forms/configuration/default/birth.ts#L121

Adding
```
template: {
  fieldName: 'informantType',
  operation: 'plainInputTransformer'
}
```

might even do the trick

File: packages/components/src/theme.ts [hunk 4]
```diff
@@ -27,3 +27,7 @@ export type ITheme = ReturnType<typeof getTheme>
 declare module 'styled-components' {
   export interface DefaultTheme extends ITheme {}
 }
+
```
Comment (rikukissa) [inline packages/components/src/theme.ts:30]: Why is this needed or was it added by accident?

File: packages/migration/utils/elasticsearch-helper.js [hunk 5]
```diff
@@ -85,3 +85,22 @@ export const searchByCompositionId = async (compositionId) => {
     return null
   }
 }
+
+export const searchByCompositionId = async (compositionId) => {
```
Comment (rikukissa) [inline packages/migration/utils/elasticsearch-helper.js:89]: Why is this needed or was it added by accident?

File: packages/client/src/views/PrintCertificate/ReviewCertificateAction.tsx (hunk 3 above)
Comment (wilrona) [reply packages/client/src/views/PrintCertificate/ReviewCertificateAction.tsx:363]: I have just applied your recommendation but I have not had the same result.
